x-pack/platform/plugins/shared/fleet/public/applications/integrations/sections/epm/screens/installed_integrations/components/installed_integration_action_menu.tsx

```diff
@@ -75,16 +81,38 @@ export const InstalledIntegrationsActionMenu: React.FunctionComponent<{
       <EuiContextMenuItem
         key="uninstall"
         icon="trash"
-        disabled={!hasUninstallableIntegrations}
+        disabled={hasUninstallableIntegrations}
         onClick={openUninstallModal}
       >
-        <FormattedMessage
-          id="xpack.fleet.epmInstalledIntegrations.bulkUninstallButton"
-          defaultMessage={'Uninstall {count, plural, one {# integration} other {# integrations}}'}
-          values={{
-            count: selectedItems.length,
-          }}
-        />
+        {hasUninstallableIntegrations ? (
+          <EuiToolTip
+            position="right"
+            content={
+              <FormattedMessage
+                id="xpack.fleet.epmInstalledIntegrations.uninstallDisabledTooltip"
+                defaultMessage="Cannot uninstall integrations that are attached to agent policies"
```

Comment:
I think we generally use contractions ([source](https://eui.elastic.co/docs/content/language/#contractions)).

```suggestion
                defaultMessage="Can't uninstall integrations that are attached to agent policies"
```

I missed this in one spot in https://github.com/elastic/kibana/pull/224334, but it's probably okay to leave it there. (Apologies, I'm new to reviewing UI text. 🙃)